docs/IRISMapCube.rst

```diff
@@ -4,72 +4,212 @@
+To initialise an IRISMapCube, we need to open an IRIS level 2 SJI FITS file. Such data can be downloaded from the IRIS_. The level 2 FITS files can be loaded with the function ``read_iris_sji_level2_fits``. This function loads a FITS file into an IRISMapCube object.
 
-Let's assume that we will call our fits file ``my_fits_file`` and IRISMapCube object as
-``my_cube``: ::
+Assuming we have a FITS file ``my_fits_file``, an IRISMapCube can be initialised in the following way: ::
 
     >>> from irispy.sji import read_iris_sji_level2_fits
     >>> my_cube = read_iris_sji_level2_fits(my_fits_file)
 
 If you don't have a lot of RAM memory or if you are loading a huge file, we recommend to
-use the memmap kwarg. By using it, you will only load what you need to run but some
-methods that requires all the file will not be accessible. You can use the memmap
-kwarg by doing: ::
+use the ``memmap=True`` keyword. By using it, you will only load the data when needed. However,
+some methods that require all data in memory will not be accessible. You can use memmap
+by doing: ::
 
     >>> from irispy.sji import read_iris_sji_level2_fits
     >>> my_cube = read_iris_sji_level2_fits(my_fits_file, memmap=True)
 
+Once we have our ``IRISMapCube`` object, we can quickly inspect it. For that, we can use the ``__repr__`` property of our object, which shows some metadata. This property is called just by writing the name of our
+object in the command line: ::
+
+    >>> my_cube
+    IRISMapCube
+    -----------
+    Observatory:		 IRIS
+    Instrument:			 SJI
+    Bandpass:			 1330.0
+    Obs. Start:			 2018-04-26T23:07:22.780000
+    Obs. End:			 2018-04-27T01:39:47.122000
+    Instance Start:		 2018-04-26T23:07:22.880000
+    Instance End:		 2018-04-27T01:36:40.490000
+    Total Frames in Obs.:	 49
+    IRIS Obs. id:		 3690015104
+    IRIS Obs. Description:	 Very large sit-and-stare 0.3x175 1s  C II   Si IV   Mg II h/k Deep x
+    Cube dimensions:		 [  49. 1095. 1018.] pix
+    Axis Types:			 ('time', 'custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon')
+
+Here, we find some information about our object, such as when the observation was made,
+how many frames it contains, its dimensions, etc ...
+
 Class Structure
 ---------------
 
-So now, ``my_cube`` is an IRISMapCube object and we can access to a lot of information like:
+Our IRISMapCube object, here called ``my_cube``, is now created and inspected, we can look into its structure.
+
+Data
+^^^^
+
+The ``data`` attribute provides direct access to the data arrays. These values have a unit
+(see :ref:`Unit`), an uncertainty (see :ref:`Uncertainty`), and can be scaled or not
+(see :ref:`Scaled_Value`). A quick representation of these data can be seen by entering: ::
+
+    >>>my_cube.data
+
+This will show a summary start and and of the data array. It such printouts it is common to see many ``nan`` values. This is because IRIS SJI images are often padded, and these values are masked (see :ref:`Mask`).
```

Comment:
We should remove reference to ```nan``` values as PR #93 will undo the assignment of ```nan``` values.  Perhaps this sentence could be something like:
```Note that IRIS SJI images are often padded with non-physical values, e.g. -200.  These pixels are identified by the IRISMapCube :ref:`Mask`.```